packages/kit/types.internal.d.ts

```diff
@@ -43,6 +43,7 @@ export type ValidatedConfig = {
 			pages: string[];
 		};
 		target: string;
+		vite: () => {};
```

Comment:
Is this not used for user-facing types, so there's no need to say that `vite` may be an object instead of a function that returns one?

Reply:
It's not, it's just for validated config. Forgot to update the types for user-facing config. Done: f8f7e707b92b1863507f2c65ccd700009da65bbf